### DISCUSSION TITLE
YesPlayMusic - Windows 不彈窗方案

### DISCUSSION THREAD

**Discussion by pan93412** [qier222/YesPlayMusic#1238] [General]
<h1 align="center">升級到 0.4.5 就不用這個方案了！</h1>

----

## 安裝

1. 從右方 OneDrive 連結下載想要的修正版：<https://1drv.ms/u/s!AiC-9Gl0ltrag9ovTSLkXlZ-Z59c_Q?e=wGqxC0>
2. 檢查下載到之修正版的 SHA256 是否符合對應 `.sha256` 檔案內的雜湊值。
3. 放在你絕對不會誤刪的資料夾（如 `C:\`)。
4. 遵循影片步驟安裝（後面步驟主要是美化捷徑，覺得無妨可不操作）。

https://user-images.githubusercontent.com/28441561/149619963-025d8dc8-ddf8-4c9e-8201-b0818cf46906.mp4

## 自行打包

OneDrive 的包也是用 <https://github.com/pan93412/ypm-win-build> 打包的，
因此假如你使用的是 macOS / Linux，可以使用這個腳本自己打包。

## 更新日誌

- v0.4.4-1 (Apr 4): 自動打包，加上了 README 和 `start_silently.bat`（開啟不彈黑色 cmd 視窗）
- v0.4.3 (Jan 15): First Release
 
## Credit

基於 https://github.com/qier222/YesPlayMusic/issues/1145#issue-1090058730 @memorydream 的替代解決方案。

**Comment by inspire-lts** [comment]
@pan93412  你好，影片不能播放额

**Reply by pan93412** [reply]
試試看：

https://user-images.githubusercontent.com/28441561/150710882-aba35d5c-8037-4c71-8ddb-8004c65de4ae.mp4

**Comment by MaxWilliams520** [comment]
可以写个vbs，隐藏cmd窗口

**Comment by GrassBlock1** [comment]
如果您所在的网络不方便访问Onedrive个人版及相关服务，也可以使用以下的蓝奏link：
https://gblock.lanzoup.com/iiVcYzsx9bg
（完全照原样提供）

**Comment by Prepanpan** [comment]
其实可以把cmd窗口改成PowerShell窗口，这样弹出的提示是带颜色的好看些

**Comment by Mint328** [comment]
我这下载了档案按照视频的方法操作后还是会出现不能播放的问题，只是不会弹出报错弹窗。。

**Comment by Prepanpan** [comment]
这个只是解决报错的问题啦......




------------------&nbsp;原始邮件&nbsp;------------------
发件人: ***@***.***&gt;; 
发送时间: 2022年3月15日(星期二) 晚上10:20
收件人: ***@***.***&gt;; 
抄送: ***@***.***&gt;; ***@***.***&gt;; 
主题: Re: [qier222/YesPlayMusic] YesPlayMusic - Windows 不彈窗方案 (Discussion #1238)





 
我这下载了档案按照视频的方法操作后还是会出现不能播放的问题，只是不会弹出报错弹窗。。
 
—
Reply to this email directly, view it on GitHub, or unsubscribe.
Triage notifications on the go with GitHub Mobile for iOS or Android.

**Reply by pan93412** [reply]
其實正常來講，這麼做之後，也應該可以透過 UNM 播放其他音源的音樂。他的情況可能需要進一步釐清。

**Comment by Prepanpan** [comment]
同意。顺带着吐槽一下UNM真的经常搞错曲子lollll



------------------&nbsp;原始邮件&nbsp;------------------
发件人: ***@***.***&gt;; 
发送时间: 2022年3月15日(星期二) 晚上11:26
收件人: ***@***.***&gt;; 
抄送: ***@***.***&gt;; ***@***.***&gt;; 
主题: Re: [qier222/YesPlayMusic] YesPlayMusic - Windows 不彈窗方案 (Discussion #1238)





 
其實正常來講，這麼做之後，也應該可以透過 UNM 播放其他音源的音樂。他的情況可能需要進一步釐清。
 
&mdash;
Reply to this email directly, view it on GitHub, or unsubscribe.
Triage notifications on the go with GitHub Mobile for iOS or Android.

**Comment by 1807086829** [comment]
打开播放器之后  后台的cmd窗口可以关闭吗

**Reply by pan93412** [reply]
0.4.4-1 裡面有個 `start_sliently.bat`，把 `start.bat` 改指到這個檔案，就不會彈出 cmd 視窗了。

**Comment by Prepanpan** [comment]
我是不太会关啦 我感觉是不能关了


发件人: 1807086829
发送时间: 2022年4月9日 15:41
收件人: qier222/YesPlayMusic
抄送: Prepanpan; Comment
主题: Re: [qier222/YesPlayMusic] YesPlayMusic - Windows 不彈窗方案 (Discussion #1238)

打开播放器之后 后台的cmd窗口可以关闭吗

**Reply by pan93412** [reply]
0.4.4-1 裡面有個 `start_sliently.bat`，把 `start.bat` 改指到這個檔案，就不會彈出 cmd 視窗了。
